components/email/test/editorial_signup.coffee

```diff
@@ -86,29 +93,6 @@ describe 'EditorialSignupView', ->
         ARTICLE: channel_id: '123'
         ARTSY_EDITORIAL_CHANNEL: '123'
         SUBSCRIBED_TO_EDITORIAL: false
-        MEDIUM: 'social'
       $subscribe = $('<div></div>')
       @view.onSubscribe({currentTarget: $subscribe})
       $($subscribe).hasClass('loading-spinner').should.be.false()
-
-  describe 'CTA popup', ->
-
-    it 'is hidden if an auction reminder is visible', ->
-      @EditorialSignupView.__set__ 'sd',
-        ARTICLE: channel_id: '123'
-        ARTSY_EDITORIAL_CHANNEL: '123'
-        SUBSCRIBED_TO_EDITORIAL: false
-        MEDIUM: 'social'
-      @view.initialize()
-      _.defer ->
-        @setupCTAWaypoints.called.should.not.be.ok()
-
-    it 'is displayed if there arent any auction reminders', ->
-      @EditorialSignupView.__set__ 'sd',
-        ARTICLE: channel_id: '123'
-        ARTSY_EDITORIAL_CHANNEL: '123'
-        SUBSCRIBED_TO_EDITORIAL: false
-        MEDIUM: 'social'
-      @view.initialize()
-      mediator.trigger 'auction-reminders:none'
-      @setupCTAWaypoints.called.should.be.ok()
```

Comment:
hmm maybe we can find a way to get these tests back in? Or are the irrelevant now? happy to pair on this part!